### ISSUE TITLE
some kind of TestMode missing

### ISSUE THREAD

**Issue by Virtex7** [pekkaroi/bldc-drive#5]
I have started to read the code and looked at the gui as well but i think there is some kind of test mode missing, where you could just start the motor at very slow speed to test it without any encoder. 

if I understand the code good enough, i will add such a mode to the software. i think that this will make development easier for beginners or people who have trouble with the encoder....

**Comment by pekkaroi** [comment]
You are correct, there is no such mode and I agree, it might be useful in the beginning. Feel free to implement that. Personally I did that by modifying the code forcing the duty cycle to some small value, but that is not very elegant solution. :)
